### ISSUE TITLE
Retraining r18 model of DBNet can't converge

### ISSUE THREAD

**Issue by msnqqer** [open-mmlab/mmocr#349]
I run the scripts to retrain r18 model of dbnet using ICDAR2015 trainset, but the loss_prob couldn't decrease, which was always about 2.85 until the training finished. I used the default config file: dbnet_r18_fpnc_1200e_icdar2015.py.  Are there some suggestions about the configs? thanks.  
BTW: in the training logs of the r18 model in the DBNet page, the model config file is dbnet_r18_fpnc_sbn_1200e_icdar2015.py, how about the difference between dbnet_r18_fpnc_sbn_1200e_icdar2015.py and the default dbnet_r18_fpnc_1200e_icdar2015.py? thanks

**Comment by gaotongxiao** [comment]
We only support `mmdet==2.11.0` for now, `mmdet>=2.12.0` would cause errors in weight initialization and we are fixing that.

**Comment by msnqqer** [comment]
> We only support `mmdet==2.11.0` for now, `mmdet>=2.12.0` would cause errors in weight initialization and we are fixing that.

mmdet that I use is v2.11.0

**Comment by gaotongxiao** [comment]
Our log shows that log_prob stays at ~2.85 for quite a long time, so it seems not a bug. It might just get stuck at the local optimum.

**Comment by msnqqer** [comment]
> Our log shows that log_prob stays at ~2.85 for quite a long time, so it seems not a bug. It might just get stuck at the local optimum.

Are there some suggestions for this problem? I have retrained the model several times, but the loss_prob still can't decrease.

**Comment by gaotongxiao** [comment]
How does your model's performance change on the validation set? And how many GPUs are you using?

**Comment by msnqqer** [comment]
> How does your model's performance change on the validation set? And how many GPUs are you using?

The hmean-iou stays at about 0. I just used one GPU.

**Comment by gaotongxiao** [comment]
We used 24 GPUs to train dbnet so it converged much faster. You can try another config which uses the model pretrained on Synthtext and achieved ~0.6 hmean-iou after the first epoch. That might converge faster.

**Comment by gaotongxiao** [comment]
I'm closing this issue now

**Comment by liuheng92** [comment]
> We used 24 GPUs to train dbnet so it converged much faster. You can try another config which uses the model pretrained on Synthtext and achieved ~0.6 hmean-iou after the first epoch. That might converge faster.

hi, Can you share the config about training on 24 GPU? My model can't get converging either when I set bigger batch.

**Comment by gaotongxiao** [comment]
@liuheng92 We are unable to find the 24-GPU log for now. But based on our recent experiments, DBNet can be trained on 1 GPU with the default config, and still converge within 1200 epochs.

**Comment by liuheng92** [comment]
> @liuheng92 We are unable to find the 24-GPU log for now. But based on our recent experiments, DBNet can be trained on 1 GPU with the default config, and still converge within 1200 epochs.

Thanks for your reply. This algrithom can converge by using default config, but when I use multi-GPU and set bigger batch,  I can't get the same results. So I am curious about how you trained on multi-GPUs.

**Comment by OneYearIsEnough** [comment]
same problem, but when I change the optimization function from 'sgd' to 'Adam', is has been solved, maybe you can use this method to have a try...🤣
<img width="1395" alt="image" src="https://user-images.githubusercontent.com/81819512/177915658-c90aae09-b4fe-41e2-a255-312912ea63cd.png">
